test/unit/views/StatementPageSpec.scala

```diff
@@ -43,6 +45,8 @@ class StatementPageSpec extends WordSpec with Matchers {
   val messagesApi: MessagesApi = app.injector.instanceOf[MessagesApi]
   implicit val messages: Messages = messagesApi.preferred(fakeRequest)
 
+  implicit val timeout: Timeout = Timeout(Duration(2, SECONDS))
+
```

Comment:
Are you setting this just to be explicit?

Reply:
@matthewmascord It's required by the `contentAsString` helper.